dq-game/src/utils/socket_connection.py

```diff
@@ -15,9 +15,9 @@
 
 # TODO: Separate client and server socket connections in dif files
 class SocketConnection(EventHandler):
-    def __init__(self, port, socket=socket.socket(socket.AF_INET, socket.SOCK_STREAM)):
+    def __init__(self, port):
         self.port = port
-        self.tcpsock = socket
+        self.tcpsock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
this `socket` is the one imported as a library ?

Reply:
yes